Aztec/Classes/Converters/NSAttributedStringToNodes.swift

```diff
@@ -487,83 +535,133 @@ private extension NSAttributedStringToNodes {
+            font.containsTraits(.traitItalic) else {
+                return nil
         }
 
-        if font.containsTraits(.traitItalic) {
-            let representation = attributes[ItalicFormatter.htmlRepresentationKey] as? HTMLElementRepresentation
-            let node = representation?.toNode() ?? ElementNode(type: .i)
+        let element: ElementNode
 
-            nodes.append(node)
+        if let representation = attributes[ItalicFormatter.htmlRepresentationKey] as? HTMLRepresentation,
+            case let .element(representationElement) = representation {
+
+            element = representationElement.toElementNode()
+        } else {
+            element = ElementNode(type: .em)
         }
 
-        return nodes
+        return element
     }
 
-
     /// Extracts all of the Link Elements contained within a collection of Attributes.
     ///
-    private func processLinkStyle(in attributes: [String: Any]) -> [ElementNode] {
+    private func processLinkStyle(in attributes: [String: Any]) -> ElementNode? {
         guard let url = attributes[NSLinkAttributeName] as? URL else {
-            return []
+            return nil
         }
 
-        let representation = attributes[LinkFormatter.htmlRepresentationKey] as? HTMLElementRepresentation
-        let node = representation?.toNode() ?? ElementNode(type: .a)
-        node.updateAttribute(named: "href", value: url.absoluteString)
+        let element: ElementNode
+
+        if let representation = attributes[LinkFormatter.htmlRepresentationKey] as? HTMLRepresentation,
+            case let .element(representationElement) = representation {
+
+            element = representationElement.toElementNode()
+        } else {
+            element = ElementNode(type: .a)
+        }
 
-        return [node]
+        element.updateAttribute(named: "href", value: .string(url.absoluteString))
+
+        return element
     }
 
 
     /// Extracts all of the Strike Elements contained within a collection of Attributes.
     ///
-    private func processStrikethruStyle(in attributes: [String: Any]) -> [ElementNode] {
+    private func processStrikethruStyle(in attributes: [String: Any]) -> ElementNode? {
         guard attributes[NSStrikethroughStyleAttributeName] != nil else {
-            return []
+            return nil
         }
 
-        let representation = attributes[StrikethroughFormatter.htmlRepresentationKey] as? HTMLElementRepresentation
-        let node = representation?.toNode() ?? ElementNode(type: .strike)
+        let element: ElementNode
+
+        if let representation = attributes[StrikethroughFormatter.htmlRepresentationKey] as? HTMLRepresentation,
+            case let .element(representationElement) = representation {
+
+            element = representationElement.toElementNode()
+        } else {
+            element = ElementNode(type: .strike)
+        }
 
-        return [node]
+        return element
```

Comment:
(Same comment as below!) can be simplified by just returning `toElementNode()` or `ElementNode(type: .strike)`, no need to use the var!